web/multinode/src/app/store/bandwidth.ts

```diff
@@ -55,9 +56,8 @@ export class BandwidthModule implements Module<BandwidthState, RootState> {
      * @param ctx - context of the Vuex action.
      */
     public async fetch(ctx: ActionContext<BandwidthState, RootState>): Promise<void> {
-        const selectedSatelliteId = ctx.rootState.nodes.selectedSatellite ? ctx.rootState.nodes.selectedSatellite.id : null;
+        const selectedSatelliteId = ctx.rootState.nodes.selectedSatellite ? ctx.rootState.nodes.selectedSatellite.id : ctx.rootState.nodes.trustedSatellites[0].id;
```

Comment:
Can we guarantee that always `trustedSatellites` will have at least one element?

Reply:
If there is a node there should be a satellite. Let me check so that I can make it better.

Reply:
I see, my fault!